core/src/main/java/io/undertow/server/handlers/proxy/mod_cluster/ModClusterStatus.java

```diff
@@ -62,7 +62,7 @@ interface LoadBalancer {
         /**
          * @deprecated Use {@link LoadBalancer#getMaxRetries()}.
          */
-        @Deprecated
+        @Deprecated(since="2.3.0", forRemoval=true)
```

Comment:
This one was also added by UNDERTOW-1315, fixed at version 2.0.4.Final, forRemoval can be true.